doc/fairness.ipynb

```diff
@@ -115,11 +115,162 @@
    "source": [
```

Comment:
![image](https://user-images.githubusercontent.com/1019791/65389012-9b5a8100-dd51-11e9-9432-e9350f1761d8.png)

the latex does not seem to render on the page? might be a github rendering issue but worthwhile to check.

Reply:
it seems to be a github issue as when I `make docs` it renders fine